metadata-models/src/main/pegasus/com/linkedin/dashboard/DashboardInfo.pdl

```diff
@@ -100,6 +100,19 @@ record DashboardInfo includes CustomProperties, ExternalReference {
   }
   datasetEdges: optional array[Edge]
 
+  /**
+   * Dashboards included by this dashboard.
+   * Some dashboard entities (e.g. PowerBI Apps) can contain other dashboards.
+   */
+  @Relationship = {
+    "/*": {
+      "name": "Contains",
```

Comment:
I vote on new relationship name, even something ugly like `DashboardContainsDashboard`. In general I think the potential downsides of shared relationship names (inability to distinguish between the two for some future use case) outweigh the benefits (shared logic)